consensus/raft/raft.go

```diff
@@ -686,3 +695,47 @@ func find(s []string, elem string) bool {
 	}
 	return false
 }
+
+func (rw *raftWrapper) observePeers() {
+	obsCh := make(chan hraft.Observation, 1)
+	defer close(obsCh)
+
+	if !sixtyfour {
+		// 32-bit systems don't support observers
```

Comment:
I am not sure what was the issue here.

https://github.com/hashicorp/raft/blob/dcbf3052ba602706d2a0baa7c51a4f23c597dd7f/observer.go#L40-L43 makes me think that it is fixed
There is a PR with similar problem containing more details.

Reply:
Do you have a raspberry Pi you can test with?

Reply:
No, but I can order one

Reply:
I think you can just build for 32 bit architecture and run it

Reply:
I think it works. 

I commented this sixtyfour check.
installed cluster like this `env GOOS=linux GOARCH=386 make install`
ran a cluster with two peers, remove one of them with `peers rm`
other peer observed it successfully and removed it from peerstore, just like 64 bit.